cob_twist_controller/include/cob_twist_controller/cob_twist_controller_data_types.h

```diff
@@ -200,14 +208,17 @@ struct TwistControllerParams
         controller_interface(VELOCITY_INTERFACE),
         integrator_smoothing(0.2),
 
-        numerical_filtering(false),
+        singularity_avoidance(SA_SR),
         damping_method(MANIPULABILITY),
         damping_factor(0.2),
         lambda_max(0.1),
         w_threshold(0.005),
         beta(0.005),
         eps_damping(0.003),
         eps_truncation(0.001),
+        damping_delta(0.1),
+        damping_gain(0.02),
+        damping_slope(0.05),
```

Comment:
@ipa-bfb 
These parameters are used within JointSingularityAvoidance, right....shouldn't they then be named ```jsa_delta```, ```jsa_gain```, ```jsa_slope```?